plugins/lsp/conform.nix

```diff
@@ -0,0 +1,90 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.conform;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.conform =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "conform.nvim";
+
+      package = helpers.mkPackageOption "conform.nvim" pkgs.vimPlugins.conform-nvim;
+
+      # Map of filetype to formatters
+      formattersByFt =
+        helpers.defaultNullOpts.mkNullable types.attrs
+        "see documentation"
+        ''
+          Map of filetype to formatters
+          lua = { "stylua" },
+          Conform will run multiple formatters sequentially
+          python = { "isort", "black" },
+          Use a sub-list to run only the first available formatter
+          javascript = { { "prettierd", "prettier" } },
+          Use the "*" filetype to run formatters on all filetypes.
+          ["*"] = { "codespell" },
+          Use the "_" filetype to run formatters on filetypes that don't
+          have other formatters configured.
+          ["_"] = { "trim_whitespace" },
+        '';
+      # If this is set, Conform will run the formatter on save.
+      # It will pass the table to conform.format().
```

Comment:
```suggestion
        '';

      # If this is set, Conform will run the formatter on save.
      # It will pass the table to conform.format().
```